contributing.md

```diff
@@ -224,21 +223,9 @@ If you add a new file of mixins, for example for a new *atom*, make sure you add
 
 ### React component
 
-Each React component has its own subdirectory `packages/bpk-component-<component-name>`, where the npm module is stored. For it to be properly published, create a `package.json` starting at version `0.0.0` and take inspiration from existing packages in terms of structure. We use [CSS Modules](https://github.com/css-modules/css-modules) along with [BEM](http://getbem.com/) to prevent collisions and accidental overwrites in CSS.
-
-#### Storybook
-
-Create a `stories.js` file and use it to list all the different possible states of your component. Check out Storybook's documentation for more details, and also check out the `stories.js` files of existing components.
-
-> Make sure you import your new component's stories into Storybook by adding it to `.storybook/config.js`.
-
-#### Testing
-
-React components need to be properly unit tested. We use [jest](https://facebook.github.io/jest/)'s snapshot testing, but depending on the complexity of your component, you may want to add additional tests.
-
-#### Readme
+Use `npm run new-component` to create a new skeleton React component. Once this is created, use existing components for code style inspiration.
```

Comment:
Shall we rename to `create-component`. It seems more logical to name it using a verb rather than a pro-noun!